docs/area-owners.md

```diff
@@ -48,58 +48,54 @@ Below table shows the combined area owners on this repository:
 | area-Tracing-mono | @lambdageek | |
 | area-Performance-mono | @SamMonoRT | |
 | **System contract assemblies** | | |
-| area-System.AppContext | @safern @Anipik | |
-| area-System.Buffers | @layomia @JeremyKuhne @ahsonkhan | |
+| area-System.Buffers | @layomia @JeremyKuhne | |
 | area-System.CodeDom | @buyaa-n @krwq | |
-| area-System.Collections | @safern | </ul>Excluded:<ul><li>System.Array -> System.Runtime</li></ul> |
-| area-System.ComponentModel | @maryamariyan @safern | |
+| area-System.Collections | @eiriktsarpalis @layomia | </ul>Excluded:<ul><li>System.Array -> System.Runtime</li></ul> |
+| area-System.ComponentModel | @JeremyKuhne @safern | |
+| area-System.ComponentModel.Composition | @maryamariyan @ViktorHofer | |
 | area-System.ComponentModel.DataAnnotations | @lajones @ajcvickers | |
 | area-System.Composition | @maryamariyan @ViktorHofer | |
 | area-System.Configuration | @maryamariyan @safern | |
-| area-System.Console | @wtgodbe | |
+| area-System.Console | @carlossanlop @eiriktsarpalis | |
 | area-System.Data | @ajcvickers @cheenamalhotra @david-engel | <ul><li>Odbc, OleDb - [@saurabh500](https://github.com/saurabh500)</li></ul> |
 | area-System.Data.SqlClient | @cheenamalhotra @david-engel @karinazhou @JRahnama | Archived component - limited churn/contributions (see https://devblogs.microsoft.com/dotnet/introducing-the-new-microsoftdatasqlclient/) |
-| area-System.Diagnostics | @tommcdon @wtgodbe @krwq | <ul><li>System.Diagnostics.EventLog - [@Anipik](https://github.com/Anipik)</li></ul> |
-| area-System.Diagnostics.Process | @wtgodbe @krwq | |
-| area-System.Diagnostics.Tracing | @noahfalk @tommcdon @Anipik | Packages:<ul><li>System.Diagnostics.DiagnosticSource</li><li>System.Diagnostics.PerformanceCounter - [@Anipik](https://github.com/Anipik)</li><li>System.Diagnostics.Tracing</li><li>System.Diagnostics.TraceSource - [@Anipik](https://github.com/Anipik)</li></ul><br/> |
+| area-System.Diagnostics | @tommcdon @krwq | <ul><li>System.Diagnostics.EventLog - [@Anipik](https://github.com/Anipik)</li></ul> |
+| area-System.Diagnostics.Process | @adsitnik @eiriktsarpalis | |
```

Comment:
Adams GitHub alias is incorrect. Should be @adamsitnik (same elsewhere in this doc).